modules/client/src/addressList/internal/client/decoding.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from "../../../client-common";
 import { AVAILABLE_FUNCTION_SIGNATURES } from "../constants";
 import { IClientAddressListDecoding } from "../../interfaces";
-import { WhitelistVoting__factory } from "@aragon/core-contracts-ethers";
+import { AllowlistVoting__factory } from "@aragon/core-contracts-ethers";
```

Comment:
Here we have some mismatch, because the SDK calls this plugin AddressList whereas the solidity project calls it AllowList. We agreed to name it like the first in some of the coordinations channels, I guess we'll need a follow-up here

Reply:
Yes, this is one thing that is confusing. Secondly, WhitelistVoting is the base of AllowlistVoting and ERC20Voting.
We cannot name it here AddressList because this is imported from the contracts